includes/admin/admin-filters.php

```diff
@@ -136,7 +136,13 @@ function give_import_core_settings_merge_pages( $json_to_array, $type ) {
 function give_import_core_settings_merge_image_size( $json_to_array, $type ) {
 	if ( 'merge' === $type ) {
 		// Featured image sizes import under Display Options > Post Types > Featured Image Size.
-		if ( 'enabled' === $json_to_array['form_featured_img'] ) {
+		if (
+			! empty( $json_to_array['form_featured_img'] )
+			&&
+			! empty( $json_to_array['featured_image_size'] )
```

Comment:
@raftaar1191 why are these on separate lines?

Reply:
@DevinWalker 

It looks good will review the code

Reply:
cleaning up the formatting @DevinWalker

Reply:
@DevinWalker Code formatting updated